.github/workflows/build-push-docker.yml

```diff
@@ -27,4 +27,4 @@ jobs:
         run:  docker build -t obscuro_enclave -f dockerfiles/enclave_local_build.Dockerfile .
 
       - name: Test
-        run: go test -v -tags docker -run TestDocker ./...
+        run: DOCKER_TEST_ENABLED=true go test -v -run TestDocker ./...
```

Comment:
Isn't it better practice to use an `env:` block here?

Reply:
Think for one var I wouldn't bother, but I can change that 👍